main.js

```diff
@@ -0,0 +1,28 @@
+const tweeter = Tweeter()
+const renderer = Renderer()
+
+const post = function(){
+    tweeter.addPost($("#input").val())
+    $("#input").val("")
+    renderer.renderPosts(tweeter.getPosts())
+}
+
+$("body").on("click",".delete",function(){
+    tweeter.removePost($(this).closest(".post").data().id)
+    renderer.renderPosts(tweeter.getPosts())
+})
+
+$("body").on("click",".add",function(){
+    let text = $(this).closest(".post").find("input").val()
+    let p = $(this).closest(".post").data().id
```

Comment:
even though `p` is short. Short is not always better. You want your code to be easy to read. It would be better just to write `postId` so people can understand it better.